autonomy/chain/tx.py

```diff
@@ -79,19 +76,28 @@ def should_retry(error: str) -> bool:
 
 def should_reprice(error: str) -> bool:
     """Check an error message to check if we should reprice the transaction"""
-    return "FeeTooLow" in error or "ReplacementNotAllowed" in error
+    error = error.lower()
+    return "feetoolow" in error or "is too low for the next block" in error
```

Comment:
Seems a potentially disruptive change. Are we sure about this change in the message? Consistent in all chains ?